JDXConverter.py

```diff
@@ -557,7 +600,118 @@ def startCommandLine(dataBaseFileName='MoleculesInfo.csv'):
+    JDXfilename=''
+    listOfJDXFileNames=list()
+    AllSpectra=[]
+    individual_spectrum=[]
+
+    MoleculeNames=list()
+    DataBase_data_holder=[]
+
+    #Reading the CSV file for database
+    print(f"LOADING Information from {dataBaseFileName}")
+    DataBase_data_holder = readFromLocalCSVDatabaseFile(dataBaseFileName)
+
+    #Starting text for the application , also instructions for the User to start
+    print("WELCOME!")
+    print("If a molecule name has a comma in it (e.g. 1,3-pentadiene) or any other input has a comma in it, we recommend using an _ (e.g. 1_3-pentadiene) since this information is stored in a comma separated value file.")
+    print("ENTER A MOLECULE NAME, OR MULTIPLE MOLECULE NAMES. Separate multiple names using ';'.")
+    while True:
+        #Taking continuous input of molecules from the user
+        moleculeName = input()
+        if(moleculeName == 'END'): break
+        if(';' in moleculeName):
+            MoleculeNames = takeInputAsList(moleculeName)
+        else:
+            MoleculeNames.append(moleculeName)
+        
+        #Getting the Data list if the Molecule name exists inside the database CSV file
+        molecule_meta_data_from_database = getDataIfMoleculeExists(DataBase_data_holder , moleculeName) #getDataIfMoleculeExists
+
+        #Now We will check if the molecule's data exist inside the database file
+        if (len(molecule_meta_data_from_database) != 0):
+            #Molecule FOUND inside the DATABASE FILE
+            #TODO: Populating these variables can be a function itself
+            
+            filename = molecule_meta_data_from_database[3].strip(' ')
+            if(filename != ''):
+                if(filename in os.listdir(defaultJDXFilesLocation)):
+                    individual_spectrum.extend(getSpectrumDataFromLocalJDX([JDXfilename]))
+                else:
+                    #This line will get all the data from online along with the individual spectrum data for the molecule. However we will only use the Spectrum data in this case
+                    spectrum_data,molecular_formula,molecular_weight,electron_number,knownMoleculeIonizationTypeOnline, knownIonizationFactorRelativeToN2Online, SourceOfFragmentationPatternOnline, SourceOfIonizationDatumOnline = getMetaDataForMoleculeFromOnline(molecule_meta_data_from_database,moleculeName)
+                    individual_spectrum.extend(spectrum_data)
+            
+            #This block will populate the necessary variables with the metadata from the database CSV file
+            ENumber = int(molecule_meta_data_from_database[1])
+            MWeight = float(molecule_meta_data_from_database[2])
+            knownMoleculeIonizationType = molecule_meta_data_from_database[4]
+            knownIonizationFactorRelativeToN2 = molecule_meta_data_from_database[5]
+            SourceOfFragmentationPattern = molecule_meta_data_from_database[6]
+            SourceOfIonizationDatum = molecule_meta_data_from_database[7]
+

```

Comment:
The below region of code (maybe 20 or 30 lines) has good commenting about the flow